vulture/core.py

```diff
@@ -98,13 +100,15 @@ class Item(object):
     """
     Hold the name, type and location of defined code.
     """
-    def __init__(self, name, typ, filename, lineno, size=1, message=''):
+    def __init__(self, name, typ, filename, lineno, size=1, message='',
+                 confidence=0):
```

Comment:
We have to pass in a default value here just to show that this is a non-optional argument because we get `None` whenever we don't pass the flag. So IMHO it doesn't matters. Does it?